.travis.yml

```diff
@@ -101,4 +105,8 @@ before_script:
   
 script:
   - $TEST_COMMAND
-  - $PARAMEDIC_COMMAND --config ./pr/$PLATFORM --plugin $PARAMEDIC_PLUGIN_TO_TEST --buildName $PARAMEDIC_BUILDNAME
+  - if [[ "$ADDITIONAL_TESTS_DIR" != "" ]]; 
+      then cd $ADDITIONAL_TESTS_DIR && npm install && npm test;
```

Comment:
normally the `then` is on the if line IIRC :)

Reply:
which makes total sense and I used a few lines above... I don't like bash 
 
fixed